Src/CSharpier/SyntaxPrinter/SyntaxNodePrinters/TypeArgumentList.cs

```diff
@@ -9,13 +9,16 @@ public static class TypeArgumentList
     {
         public static Doc Print(TypeArgumentListSyntax node)
         {
+            Doc separator = node.Arguments.Count > 1
+            || node.Arguments.Any(o => o is GenericNameSyntax) ? Doc.SoftLine : Doc.Null;
```

Comment:
Odd. Why didn't this indent?

Reply:
I just checked this in 0.9.8 and it looks better now.
There is also #345 for improving some of the edge cases.